client/src/com/mirth/connect/client/ui/BrandingConstants.java

```diff
@@ -0,0 +1,42 @@
+package com.mirth.connect.client.ui;
+
+import javax.swing.ImageIcon;
+
+public class BrandingConstants {
+
+    public static final String PRODUCT_NAME = "OpenIntegrationEngine";
+
+    /*
+        Sets the title of the Administrator window
+     */
+    public static final String WINDOW_TITLE = "OpenIntegrationEngine Administrator";
+
+    public static final String ISSUE_TRACKER_LOCATION = "https://github.com/openintegrationengine/engine/issues";
+
+
+    // The URL that is opened when clicking the bottom image in Login window
+    // The one where you're asked for server URL, username, and password
+    public static final String MIRTHCORP_URL = "https://github.com/openintegrationengine";
+    public static final String MIRTHCORP_TOOLTIP = "OpenInterationEngine";
+
+
+    // The URL that is opened when clicking the image in the Top right corner of the main administrator window
+    public static final String MIRTHCONNECT_URL = "https://github.com/openintegrationengine";
+    public static final String MIRTHCONNECT_TOOLTIP = "OpenIntegrationEngine";
```

Comment:
suggestion: make names more generic

Can we rename?
 - MIRTHCORP -> COMPANY
 - MIRTHCONNECT -> PRODUCT